iis-wf/iis-wf-primary/src/test/resources/eu/dnetlib/iis/wf/primary/processing/sampledataproducer/output/report/pushgateway.json

```diff
@@ -1,51 +1,54 @@
-{"key":"processing.citationMatching.citDocReference.direct","type":"COUNTER","value":"1"}
-{"key":"processing.citationMatching.citDocReference.fuzzy","type":"COUNTER","value":"1"}
-{"key":"processing.citationMatching.citDocReference.total","type":"COUNTER","value":"2"}
+{"key":"processing.citationMatching.citDocReferences.direct","type":"COUNTER","value":"1"}
+{"key":"processing.citationMatching.citDocReferences.fuzzy","type":"COUNTER","value":"1"}
+{"key":"processing.citationMatching.citDocReferences.deduped","type":"COUNTER","value":"2"}
 {"key":"processing.citationMatching.direct.duration","type":"DURATION","value":"$(LONG_RANGE: [1000, 150000])"}
-{"key":"processing.citationMatching.doc.direct","type":"COUNTER","value":"1"}
-{"key":"processing.citationMatching.doc.fuzzy","type":"COUNTER","value":"1"}
-{"key":"processing.citationMatching.doc.total","type":"COUNTER","value":"1"}
+{"key":"processing.citationMatching.docs.direct","type":"COUNTER","value":"1"}
+{"key":"processing.citationMatching.docs.fuzzy","type":"COUNTER","value":"1"}
+{"key":"processing.citationMatching.docs.deduped","type":"COUNTER","value":"1"}
 {"key":"processing.citationMatching.fuzzy.duration","type":"DURATION","value":"$(LONG_RANGE: [1000, 900000])"}
-{"key":"processing.citationMatching.total.duration","type":"DURATION","value":"$(LONG_RANGE: [1000, 1500000])"}
```

Comment:
Let's drop `total` here (instead of renaming to `deduped`) as already justified in one of the comments above.